### PR TITLE
DOC: Fix EX02 in pandas.Series.memory_usage

### PR DESCRIPTION
**Pull request by farhanreynaldo**
- [ ] closes #xxxx
- [ ] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

Related to #27977. 

```
################################################################################
################################## Validation ##################################
################################################################################

### CODE REVIEW

File: pandas/core/series.py [hunk 1]
```diff
@@ -4195,7 +4195,7 @@ def memory_usage(self, index=True, deep=False):
         >>> s.memory_usage()
         144
         >>> s.memory_usage(deep=True)
-        260
+        252
```
Comment (MarcoGorelli) [inline pandas/core/series.py]: Are you sure it's not correct as it is? At least, during the CI process, 260 is returned: https://github.com/pandas-dev/pandas/pull/32653/checks?check_run_id=502744916

Comment (farhanreynaldo) [reply pandas/core/series.py]: It seems there's a difference in number returned on my machine and CI process as shown below:

image

Do you have any idea what cause the difference?

Comment (MarcoGorelli) [reply pandas/core/series.py]: What's the output of `pd.show_versions`?

Comment (farhanreynaldo) [reply pandas/core/series.py]: Here is the output of `pd.show_versions`

```python
>>> import pandas as pd
>>> pd.show_versions()

INSTALLED VERSIONS
------------------
commit           : fa9a45ac4dceedf7a34bf68efb11fff6e844aac0
python           : 3.7.6.final.0
python-bits      : 64
OS               : Windows
OS-release       : 10
Version          : 10.0.16299
machine          : AMD64
processor        : Intel64 Family 6 Model 61 Stepping 4, GenuineIntel
byteorder        : little
LC_ALL           : None
LANG             : en_US.UTF-8
LOCALE           : None.None

pandas           : 1.1.0.dev0+787.gfa9a45ac4
numpy            : 1.18.1
pytz             : 2019.3
dateutil         : 2.8.1
pip              : 20.0.2
setuptools       : 45.2.0.post20200209
Cython           : 0.29.15
pytest           : 5.3.5
hypothesis       : 5.5.2
sphinx           : 2.4.1
blosc            : None
feather          : None
xlsxwriter       : 1.2.7
lxml.etree       : 4.5.0
html5lib         : 1.0.1
pymysql          : None
psycopg2         : None
jinja2           : 2.11.1
IPython          : 7.12.0
pandas_datareader: None
bs4              : 4.8.2
bottleneck       : 1.3.1
fastparquet      : 0.3.3
gcsfs            : None
matplotlib       : 3.1.3
numexpr          : 2.7.1
odfpy            : None
openpyxl         : 3.0.1
pandas_gbq       : None
pyarrow          : 0.16.0
pytables         : None
pyxlsb           : None
s3fs             : 0.4.0
scipy            : 1.3.1
sqlalchemy       : 1.3.13
tables           : 3.6.1
tabulate         : 0.8.6
xarray           : 0.15.0
xlrd             : 1.2.0
xlwt             : 1.3.0
numba            : 0.48.0
````

**Comment by WillAyd** [comment]
I think for now should just stick with what CI is yielding. I don't think the 8 byte difference is worth worrying too much about, though if you see something more problematic feel free to open an issue for it.

Thanks anyway for the PR!

**Pull request closed without merging**